apps/golang/credits

```diff
@@ -0,0 +1,4 @@
+(github.com/.../...)
```

Comment:
what is this line?

Reply:
To signify that the following lines are Github links in the form of `username/repo`

Reply:
why not just add them as links directly then?